STYLEGUIDE.md

```diff
@@ -69,6 +70,32 @@ For example--
 ```
 
 
+### Enumerations and "Other"
+
+To support "other" values in an enumeration, add the value "other"
+to the enumeration type definition.  For example--
+
+```xml
+<xs:simpleType name="Color">
+  <xs:restriction base="xs:string">
+    <xs:enumeration value="blue" />
+    <xs:enumeration value="green" />
+    <xs:enumeration value="other" />
```

Comment:
For general usability, I might also add that "other" should be the last option. While I understand having the options in alphabetical order, I think the "other" might be lost on a longer list. Maybe alphabetical-with-an-other?